src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,112 @@
+package codeu.model.data;
+
+import codeu.model.util.Util;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * Class representing an activity (creating user, message, or conversation). Any creation done by a
+ * user prompts the creation of an activity object.
+ */
+public class Activity {
+  private final UUID id;
+  private final UUID ownerId;
+  private final Action action;
+  private Boolean isPublic;
+  private final Instant creation;
+  private final String thumbnail;
```

Comment:
Is there a reason you use a wrapper class boolean opposed to primitive? I'm assuming it's because an Activity object is calling the method and not a value of primitive type?

Reply:
@donnelldebnam hmmm, I didn't think that deep about it. I just thought since all the other fields are objects-- it is better to make that Boolean an object.